bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -4,49 +4,21 @@ This binding adds support to control Tapo (Copyright © TP-Link Corporation Limi
 
 ## Supported Things
 
-The following Tapo-Devices are supported
-
-### P100/P105 SmartPlug (Wi-Fi)
-
-* Power On/Off
-* Wi-Fi signal (SignalStrength)
-* On-Time (Time in seconds device is switched on)
-
-### P110/P115 EnergyMonitoring SmartPlug (Wi-Fi)
-
-* Power On/Off
-* Wi-Fi signal (SignalStrength)
-* On-Time (Time in seconds device is switched on)
-* actual PowerUsage (Watt)
-* today EnergyUsage (Wh)
-* today Runtime (Time in seconds device was on today)
-
-### L510(Series) dimmable SmartBulb (Wi-Fi)
-
-* Light On/Off
-* Brightnes (Dimmer)  0-100 %
-* ColorTemperature (Number) 2500-6500 K
-* Wi-Fi signal (SignalStrength)
-* On-Time (Time in seconds device is switched on)
-
-### L530(Series) MultiColor SmartBulb (Wi-Fi)
-
-* Light On/Off
-* Brightnes (Dimmer)  0-100 %
-* ColorTemperature (Number) 2500-6500 K
-* Color (Color)
-* Wi-Fi signal (SignalStrength)
-* On-Time (Time in seconds device is switched on)
-
-### L900/L920 LED-LightStrip (Wi-Fi)
-
-* Light On/Off
-* Brightnes (Dimmer)  0-100 %
-* ColorTemperature (Number) 2500-6500 K
-* Color (Color)
-* Wi-Fi signal (SignalStrength)
-* On-Time (Time in seconds device is switched on)
-
+The following Tapo-Devices are supported. For precise channel-description look at `channels-table` below
+
+| Type                               | Name        | Description                                 |
+|------------------------------------|-------------|---------------------------------------------|
+| SmartPlug (Wi-Fi)                  | P100        | Smart Socket                                |
+|                                    | P105        | Smart Mini Socket                           |
+| EnergyMonitoring SmartPlug (Wi-Fi) | P110        | Energy Monitoring Smart Socket              |
+|                                    | P115        | Energy Monitoring Mini Smart Socket         |
+| Dimmable SmartBulb (Wi-Fi)         | L510        | Dimmable White-Light Smart-Bulb (E27)       |
+|                                    | L610        | Dimmable White-Light Smart-Spot (GU10)      |
+| MultiColor SmartBulb (Wi-Fi)       | L530        | Multicolor Smart-Bulb (E27)                 |
+|                                    | L630        | Multicolor Smart-Spot (GU10)                |
+| MultiColor LightStrip (Wi-Fi)      | L900        | Multicolor RGB Dimmable LightStrip (5m)     |
+|                                    | L920        | Multicolor RGB-IC ColorZone LightStrip (5m) |
+|                                    | L930        | Multicolor RGBW-IC 50-Zone LightStrip (5m)  |
```

Comment:
This is really confusing as type is not a thing type but name is a thing type.
Maybe you should rename your columns.